go-header.yaml

```diff
@@ -0,0 +1,32 @@
+---
+year: 2019
+go-project: true
+project_dir: ""
+goroutine-count: 6
+show-only-first-error: true
+scope:
+  policy: "diff"
+  master-branch: "origin/master"
+rules:
+  - template: |-
+      {NSM COPYRIHT HOLDERS}// Licensed under the Apache License, Version 2.0 (the "License");
+      // you may not use this file except in compliance with the License.
+      // You may obtain a copy of the License at:
+      //
+      //     http://www.apache.org/licenses/LICENSE-2.0
+      //
+      // Unless required by applicable law or agreed to in writing, software
+      // distributed under the License is distributed on an "AS IS" BASIS,
+      // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      // See the License for the specific language governing permissions and
+      // limitations under the License.
+    path-matcher: ""
+    author-matcher: ""
+copyright-holders:
```

Comment:
Why are we specifying Copyright holders here?

Reply:
To avoid typos into copyright holders.
@edwarnicke should we use any copyright holder?

Reply:
Yes... we will definitely have other copyright holders.

Reply:
fixed